autoscheduler/frontend/src/components/Navbar/LoginButton.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { Button } from '@material-ui/core';
+import ExitToAppOutlinedIcon from '@material-ui/icons/ExitToAppOutlined';
```

Comment:
Let's just rename it to `LogoutIcon` for our purposes. Saves a little space, and it also explains what it's doing better imo

Reply:
I had no idea I could rename this. Regardless it is done.